### PR TITLE
Remove a few unused class components, convert a few components to functions

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

File: packages/desktop-client/src/components/accounts/MobileAccountDetails.js [hunk 1]
```diff
@@ -16,62 +16,61 @@ import SearchAlternate from 'loot-design/src/svg/v2/SearchAlternate';
...
-  render() {
-    const { accountName } = this.props;
-    const { text } = this.state;
+  const onChange = useCallback(
+    text => {
+      setText(text);
+      performSearch();
+    },
+    [performSearch],
+  );
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/MobileAccountDetails.js:32]: Does this still work?

Before:
`setText` and after the state changed -> `performSearch`

After:
`setText` and in parallel (same CD cycle) `performSearch` with old `text` value

File: packages/desktop-client/src/components/manager/ManagementApp.js [hunk 2]
```diff
@@ -49,170 +49,163 @@ function Version() {
...
-    if (!this.props.managerHasInitialized) {
+    if (!managerHasInitialized) {
       if (!alreadyLoading) {
-        this.props.setAppState({ loadingText: '' });
+        setAppState({ loadingText: '' });
       }
-    } else {
-      // If it's not the first time rendering, immediately show the
-      // modal since we should have cached data
-      this.showModal();
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/ManagementApp.js:78]: Why was this removed?

**Review by MatissJanis — COMMENTED** [review]
Nice

File: packages/desktop-client/src/components/manager/ManagementApp.js (hunk 2 above)
Comment (j-f1) [reply packages/desktop-client/src/components/manager/ManagementApp.js:78]: I forgot to remove it when I deleted the method definition: https://github.com/actualbudget/actual/pull/762/files#diff-5f79220dc9789de60d3ab3e513868b55f7450825ee20364de7bb6059f54fa539L95

**Review by MatissJanis — APPROVED** [review]
